app/controllers/users/personal_keys_controller.rb

```diff
@@ -26,6 +26,7 @@ def update
     def create
       user_session[:personal_key] = create_new_code
       analytics.track_event(Analytics::PROFILE_PERSONAL_KEY_CREATE)
+      Event.create(user_id: current_user.id, event_type: :new_personal_key)
```

Comment:
Would it make more sense to call this in the personal key generator so the event get created every time a new personal key is generate? This is missing the event that occurs when the personal key is created on sign up and during identity verification.

That'd look more like phone and email, where we capture those events on sign up.

Reply:
We could do that but the hacker one finding only identified when the personal key changed and that's how this PR was scoped.

Reply:
FWIW looking at the other events, we don't have 'create' events for the individual items, only 'change' events, perhaps implying that they are covered by the 'create' event for the account. This PR is consistent with that pattern.

Reply:
Well, we trigger those events on sign up as well though. IIRC there are events for confirming email and phone that show up after you create your account.

Reply:
aaahhh, then might as well handle both cases here so we don't have to revisit...

Reply:
To me these extra conditions seem redundant since they are mandatory steps vs someone requesting a new key on your account.  Moreover they aren't giving the user any useful information for security purposes and perhaps would create more confusion.

Reply:
@andrewhughey: Any thoughts on ☝️

Reply:
I'm comfortable with the scoping as-is. It addresses the H1 report and the use case described.

We don't include in the event history that a password was created or email confirmed. The only events that show up at creation time are: `Account created` and `phone confirmed`

We're taking a good look at `/account` and account management in general, so we can solve for the larger problems there. Let's keep this to addressing "changing" the key.